### ISSUE TITLE
TypeError: Cannot read property '' of undefined

### ISSUE THREAD

**Issue by freakypie** [amark/gun#598]
```
node_modules/gun/lib/radix.js:57
                        if(u !== (tmp = tree[$])){
                                            ^

TypeError: Cannot read property '' of undefined
    at Function.map (node_modules/gun/lib/radix.js:57:24)
    at node_modules/gun/lib/store.js:44:11
    at r (node_modules/gun/lib/radisk.js:41:13)
    at Object.next (node_modules/gun/lib/store.js:42:3)
    at Object.onto [as on] (node_modules/gun/gun.js:203:41)
    at Function.Gun.on.get (node_modules/gun/gun.js:826:10)
    at Object.root [as next] (node_modules/gun/gun.js:709:14)
    at Object.next (node_modules/gun/nts.js:17:20)
    at Object.onto [as on] (node_modules/gun/gun.js:203:41)
    at Function.Mesh.mesh.hear (node_modules/gun/gun.js:1948:10)
```
This happened when I was importing a bunch of data. It isn't consistent enough to crash every time or with a specific set of objects. I tried to reduce the data i was using so that I could give you a sample, but the crash was too inconsistent.

I know it can be impossible to debug something without being able to replicate it, so i'll keep trying to isolate something that will replicate this bug, but in case you can look at the stack and get an idea of what is wrong, here it is!

**Comment by fuchidahiro** [comment]
Does it always crash with the same line?

**Comment by freakypie** [comment]
Yes.

I throttled the import to delay by 0.3 seconds between PUTs and it has gone away

**Comment by fuchidahiro** [comment]
This is really odd. So maybe too many puts at the same time make it process wrong? Freaky

**Comment by freakypie** [comment]
concurrency? I dunno, i'll try to debug it

**Comment by amark** [comment]
@freakypie @fuchidahiro hmm, odd/annoying, I had this bug a few weeks ago and fixed it (I thought), so this is definitely a legit problem if it is coming up again. Quick double check: what GUN version are you on?

**Comment by freakypie** [comment]
0.9.9997

**Comment by Dletta** [comment]
@freakypie newest is 0.9.9998, there was some bugfixes in that version... might want to npm i gun@latest and give it a try again

**Comment by amark** [comment]
@freakypie @Dletta yeah, I do think `0.9.9998` was released to fix this bug from `0.9.9997`. Of course, I always assume something is still wrong... so let us know!

**Comment by amark** [comment]
@freakypie haven't heard back in a while, so I'm gonna assume the upgrade fixed this. Please re-open if still an issue.
